inno_setup.iss

```diff
@@ -20,12 +20,12 @@ AppPublisherURL={#AppURL}
 AppSupportURL={#AppURL}
 AppUpdatesURL={#AppURL}
 Compression=lzma
+; Allow selection of installation with elevation or not.
+PrivilegesRequiredOverridesAllowed=commandline dialog
 DefaultDirName={commonpf}\{#AppName}
 DefaultGroupName={#AppName}
 AllowNoIcons=yes
 LicenseFile=LICENSE.txt
-; Remove the following line to run in administrative install mode (install for all users.)
-PrivilegesRequired=lowest
```

Comment:
Results in a Windows Error of Access Denied without proper elevation with the default folder setup.

Reply:
Hey, thanks for the catch, totally forgot that default folder (Program Files) requires admin rights 😅.
Have you been able to compile the installer with those settings and test it?

Reply:
I just downloaded Inno setup, I'll try it out, though I think I got it wrong

Reply:
Alright, I will test the changes tomorrow as well.

Reply:
Turns out i did got it wrong

Reply:
I'll read some docs and make another pull request when it works!

Reply:
There we go!